src/sentry/static/sentry/app/views/groupDetails/actions.jsx

```diff
@@ -257,6 +260,22 @@ const GroupDetailsActions = createReactClass({
           onDiscard={this.onDiscard}
         />
 
+        {hasOwners && (
+          <div className="btn-group">
+            <a
```

Comment:
`Button` or is there some weird styling with `btn-group`?

Reply:
tried this and these buttons are too different ;/